orc8r/cloud/helm/orc8r/charts/nms/templates/nginx-proxy-deployment.yaml

```diff
@@ -35,10 +35,14 @@ spec:
   selector:
     matchLabels:
 {{ tuple $envAll "nms" "nginx" | include "nms.selector-labels" | indent 6 }}
+{{ include "magmalte-image-version-label" . | indent 6 }}
   template:
     metadata:
       labels:
 {{ tuple $envAll "nms" "nginx" | include "nms.selector-labels" | indent 8 }}
+{{ include "magmalte-image-version-label" . | indent 8 }}
```

Comment:
Should we use magmalte-image-version-label for nginx-proxy deployment as well? Looking at our [artifactory for helm charts](https://artifactory.magmacore.org/artifactory/helm/), we have one for orc8r and one for lte-orc8r which I presume handles it for the controller and magmalte images respectively. If so, is there a possibility that nginx-proxy and magmalte could have different chart versions?

Reply:
> Should we use magmalte-image-version-label for nginx-proxy deployment as well

No, I think we should use the `nginx-image-version-label`. I corrected the PR to reflect that.

> If so, is there a possibility that nginx-proxy and magmalte could have different chart versions?

No, both nginx deployments will use the same [helm value configuration](https://github.com/magma/magma/blob/master/orc8r/cloud/helm/orc8r/values.yaml#L105).

Being more clear, there are two nginx deployments: 
1. [One handling the public access to NMS](https://github.com/magma/magma/blob/master/orc8r/cloud/helm/orc8r/charts/nms/templates/nginx-proxy-deployment.yaml)
2. [Another handling the public access to Orc8r services](https://github.com/magma/magma/blob/master/orc8r/cloud/helm/orc8r/templates/nginx.deployment.yaml)

We expect them to have the same image-label value, though the char-version will be different, #1 will receive the NMS sub-chart version, and the #2 will receive Orc8r version.